src/Parser/DemonHunter/Vengeance/Modules/Abilities.js

```diff
@@ -162,6 +162,19 @@ class Abilities extends CoreAbilities {
         castEfficiency: {
           suggestion: true,
           recommendedEfficiency: 0.90,
+          extraSuggestion: combatant.hasTalent(SPELLS.FLAME_CRASH_TALENT.id)?<span>Line this up with <SpellLink id={SPELLS.INFERNAL_STRIKE.id} />to double stack <SpellLink id={SPELLS.SIGIL_OF_FLAME.id} /> because of the <SpellLink id={SPELLS.FLAME_CRASH_TALENT.id} /> talent.</span>:<span>Cast on cooldown for a dps increase.</span>,
+        },
+      },
+      {
+        spell: SPELLS.SIGIL_OF_FLAME,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
+        enabled: combatant.hasTalent(SPELLS.QUICKENED_SIGILS_TALENT.id),
+        cooldown: 24,
```

Comment:
This cooldown looks to be incorrect, the tooltip says it's 30 seconds.

Reply:
Ohh my bad, I hadn't seen the enabled check. Thanks for enlightening me!

It looks like the only difference with the non-talent-specific version (which is missing an inverted `enabled` property btw as Nisekoi pointed out) is the cooldown, so you could combine the two versions into one with the solution Nisekoi suggested. I personally like to exactly match in-game, which would make it:

`cooldown: 30 * (1 - combatant.hasTalent(SPELLS.GRACEFUL_SPIRIT_TALENT.id) ? 0.2 : 0),`

Then you can reduce some of the duplicate code. Yay! 🎉

Reply:
this looks like a great idea. Will implement it.